DNN Platform/Library/Services/Scheduling/DNNScheduler.cs

```diff
@@ -54,19 +54,13 @@ public class DNNScheduler : SchedulingProvider
 
         #region Private Methods
 
-        private bool CanRunOnThisServer(string servers)
+        private static bool CanRunOnThisServer(string servers)
         {
-            string lwrServers = "";
-            if (servers != null)
-            {
-                lwrServers = servers.ToLower();
-            }
-            if (String.IsNullOrEmpty(lwrServers) || lwrServers.Contains(Globals.ServerName.ToLower()))
-            {
-                return true;
-            }
-
-            return false;
+            return
+                // Scheduler should run on every server
+                string.IsNullOrWhiteSpace(servers)
+                // Check if scheduler should run on this server
+                || servers.Split(',').Contains(Globals.ServerName, StringComparer.InvariantCultureIgnoreCase);
```

Comment:
Hi @bdukes , thanks for your comment. I've made both changes (see https://github.com/janjonas/Dnn.Platform/commit/6328ac0088b5d9f0b9a78bf151d9214e3adf8aa2). I was not aware of OrdinalIgnoreCase but this definitively makes sense and add trimming is also a good idea to make the method a bit more "robust".